spec.md

```diff
@@ -0,0 +1,315 @@
+
+## Defining OCI Artifact Types
+
+As registries become content addressable distribution points, tools that pull artifacts must know if they can operate on the artifact. Artifact types are equivalent to file extensions. 
+- When users open files, the host operating system typically launches the appropriate program.
+- When users open a file, from within a program, the open dialog filters to the supported types.
+- When search or security software scan the contents of a storage solution, the software must to know how to process the different types of content.
+- When users view the contents of a storage solution, they see the textual and visual indications of the type.
+
+OCI Artifacts provides these core capabilities to [OCI distribution spec][distribution-spec] based registries.
+
+Once complete, Artifacts can be identified and visualized as the following:
+
+|Icon|Artifact|`config.mediaType`|
+|-|-|-|
+|<img src="https://github.com/opencontainers/artwork/blob/master/oci/icon/color/oci-icon-color.png?raw=true" width=30x>|[OCI Image][image-spec]|`application/vnd.oci.image.config.v1+json`|
+|<img src="https://github.com/helm/helm-www/blob/master/themes/helm/static/img/apple-touch-icon.png?raw=true" width=30x>|[Helm Chart](https://helm.sh)|`application/vnd.cncf.helm.chart.config.v1+json`|
+|<img src="https://github.com/sylabs/singularity/blob/master/docs/logos/singularity_v3.png?raw=true" width=30x>|[Singularity][singularity], by [Sylabs][sylabs]|`application/vnd.sylabs.sif.config.v1+json`|
+
+## Authoring an OCI Artifacts
+
+Authoring Artifacts involves the following steps:
+
+1. [Define a unique Artifact type](#defining-a-unique-artifact-type)
+1. Define the format for other tools to operate upon the type.
+1. Define human elements, such as an icon and localized string to be displayed to users.
+1. Optional: Register the unique `mediaType` with [IANA][iana-register-media-types].
+1. Optional: Publish as a [well-known type][def-well-known-types] for registries to consume.
+
+### Defining a Unique Artifact Type
+
+A unique artifact is similar to defining a file extension. Defining a unique Artifact allows various tools to know how to uniquely work with the type. It allows a registry to display the type, and vulnerability scanners a means to know if and how they should scan the contents.
+
+Defining a unique type involves uniqueness for computer processing, and uniqueness for humans. See [Defining Artifact Publisher Manifests](#optional-defining-artifact-publisher-manifests) for human readability.
+
+For computer processing, artifacts are defined by setting the `manifest.config.mediaType` to a globally unique value.
+
+> **Note:** The `config.mediaType` of `application/vnd.oci.image.config.v1+json` is reserved for artifacts intended to be run and instanced by [docker][docker], [containerd][containerd] and other [OCI Image][image-spec] runtimes and tool chains.
```

Comment:
I'd also add `and instances of this type MUST conform to the [OCI Image Config specification][image-config-spec]`

Reply:
Do we need this additional text here?  I'm really just trying to get people to realize that all artifacts shouldn't use the `application/vnd.oci.image.config.v1+json` for their unique types.

Reply:
I'd like to be absolutely clear that it must meet the spec, lest people accept it without validation like the artifact types